classes/autoptimizeStyles.php

```diff
@@ -343,6 +344,10 @@ public function read( $options )
                                 if ( ! empty( $minified_url ) ) {
                                     // Replace orig URL with cached minified URL.
                                     $new_tag = str_replace( $url, $minified_url, $tag );
+                                } elseif ( $minified_url === '' ) {
```

Comment:
I guess this can be an `else` without an `if`, as we have already established `$minified_url` is empty ?

Reply:
Only thing why i did this was to skip this else statment when $minified_url = false, so it would not break other code this way. (because I don't have the full picture of all the sources involved I added this as a precousion.) 
It's fine by me to change this as you suggested.